app/javascript/components/SweetPhoneInput/SweetPhoneInput.js

```diff
@@ -75,7 +75,7 @@ class SweetPhoneInput extends Component {
   onPhoneNumberChange = (value: string = '') => {
     this.setState(prevState => ({
       ...prevState,
-      phoneNumber: format(value, this.getCountryCode(), 'National'),
+      phoneNumber: new asYouType(this.getCountryCode()).input(value),
```

Comment:
asYouType is imported from this library: https://github.com/catamphetamine/libphonenumber-js

It's the same library that was already being used, but I'm just using a different function.